src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/Razor/GenerateMethodCodeActionResolver.cs

```diff
@@ -77,9 +68,9 @@ internal class GenerateMethodCodeActionResolver : IRazorCodeActionResolver
         var razorClassName = Path.GetFileNameWithoutExtension(uriPath);
         var codeBehindPath = $"{uriPath}.cs";
 
-        if (!File.Exists(codeBehindPath)
-            || razorClassName is null
-            || !code.TryComputeNamespace(fallbackToRootNamespace: true, out var razorNamespace))
+        if (!File.Exists(codeBehindPath) ||
```

Comment:
very very off in the weeds so just ignore if needed: 

I actually prefer the `||` at the start of the line. I'm assuming in general we don't though?

Reply:
I personally don't prefer it, but happy to put it to a vote. Sadly missed our chance to hold a push up competition about it a few weeks ago.

Reply:
I don't have strong preference either way. Maybe as a tiebreaker, what does the pretty formatter/style enforcer in vscode prefer? I think it likes operators at the beginning of the line. My natural inclination is to put them at the end of the line so you understand continuation is coming - kind of like in English you put hyphen at the end of the line - not at the beginning of the next line. Which contradicts VSCode formatter/style preference (I think).

Reply:
I think most c# code prefers the style in this PR, and I'm okay with conceding. I don't have a strong enough preference to change everything over it just gauging interest. Let's keep this style

Reply:
I actually prefer a mix of `||` at the end or the beginning, depending on the situation. For 'if` conditions, I prefer them at the end and for `return` or `=>` expressions, I usually prefer them at the start. However, @davidwengier convinced me long ago to be consistent with the Razor code base, and we chose to go at the end.

@alexgav: What do you mean by the VSCode formatter/style? The Roslyn C# formatter is used for C# in VS Code, so it would be the same as VS. However, if you mean to follow a different language like TypeScript/JavaScript, I don't think that makes sense as a tiebreaker to me. The ecosystem of TypeScript/JavaScript looks very different than the ecosystem of C#.